.gitignore

```diff
@@ -10,3 +10,5 @@
 
 /shims
 /coverage
+
+/node_modules
```

Comment:
Why this change?

Reply:
Vitest stores data in `node_modules` so I ignored it.

```console
$ find node_modules/
node_modules/
node_modules/.vitest
node_modules/.vitest/deps
node_modules/.vitest/deps/_metadata.json
node_modules/.vitest/deps/package.json
node_modules/.vitest/results.json
```